rustls/src/quic.rs

```diff
@@ -342,12 +343,14 @@ pub trait ClientQuicExt {
                 .all(|x| x.get_u16() >= ProtocolVersion::TLSv1_3.get_u16()),
             "QUIC requires TLS version >= 1.3"
         );
+        let ext = if (0xff00_0000..=0xff00_0020).contains(&quic_version) {
```

Comment:
Do we have a sense of what QUIC v2 drafts might use? Should we just use `> 0xff00_0000` to try and future-proof it a little?

Reply:
Future draft versions (for e.g. QUIC 2) might use yet another TLS extension number. I think we should stick to known cases.